static/js/util/util.js

```diff
@@ -191,6 +193,42 @@ export function validateProfile(profile, requiredFields, messages) {
 }
 
 /* eslint-enable camelcase */
+/*
+check that the profile is complete. we make the assumption that a
+complete profile consists of:
+  - a valid personal tab
+  - an entry for 'currently employed', and a work history entry if
+    `currently employed == 'yes'`
+*/
+export function profileComplete(profile) {
```

Comment:
cool :) I've been writing a little Go lately for some personal projects, that sort of thing seems like a common idiom in Go-land.